packages/client/src/pages/create-card/CreateCard.tsx

```diff
@@ -7,11 +7,14 @@ import { useForm } from '../../components/form/hook/useForm';
 import Input from '../../components/form/input/Input';
 import TextArea from '../../components/form/text-area/TextArea';
 
+import { CreateCardButtonWrapper } from './CreateCard.styled';
+
 const createCardSchema = z.object({
   firstName: z.string().min(1, 'First Name must be at least 1 characters long!'),
   lastName: z.string().min(1, 'Last Name must be at least 1 characters long!'),
   email: z.string().email(),
-  textarea: z.string().optional(),
+  bio: z.string().optional(),
+  changeAvatar: z.string().min(1, 'You need to provide a path image!'),
```

Comment:
Now I have a better understanding of what we want to validate.
Just one small thing, `changeAvatar` sounds to me like some action, we should use nouns, simply we can call this field `avatar` as part of our validation model.